fcrepo-serialization/src/test/java/org/fcrepo/camel/serialization/BinaryEnabledRouteTest.java

```diff
@@ -64,14 +65,14 @@ public void configure() throws Exception {
         });
         context.start();
 
-        getMockEndpoint("mock:direct:binary_file").expectedMessageCount(1);
-        getMockEndpoint("mock:direct:binary_file").expectedHeaderReceived(Exchange.FILE_NAME, "foo");
+        getMockEndpoint("mock:file:mock:direct:binary_file").expectedMessageCount(1);
+        getMockEndpoint("mock:file:mock:direct:binary_file").expectedHeaderReceived(Exchange.FILE_NAME, "foo");
```

Comment:
While this may work, it is a very strange formulation. I wouldn't expect two `mock` components here -- either you're mocking the `file` component or you're mocking the `direct` component, but not both.